gcloud/bigtable/cluster.py

```diff
@@ -317,6 +319,34 @@ def create(self):
         self._operation_id, self._operation_begin = _process_operation(
             cluster_pb.current_operation)
 
+    def update(self):
+        """Update this cluster.
+
+        .. note::
+
+            Updates the ``display_name`` and ``serve_nodes``. If you'd like to
+            change them before updating, reset the values via
+
+            .. code:: python
+
+                cluster.display_name = 'New display name'
+                cluster.serve_nodes = 3
+
+            before calling :meth:`update`.
+        """
+        request_pb = data_pb2.Cluster(
+            name=self.name,
+            display_name=self.display_name,
+            serve_nodes=self.serve_nodes,
+        )
+        # We expect a `._generated.bigtable_cluster_data_pb2.Cluster`.
+        cluster_pb = self._client._cluster_stub.UpdateCluster(
+            request_pb, self._client.timeout_seconds)
+
+        self._operation_type = 'update'
+        self._operation_id, self._operation_begin = _process_operation(
+            cluster_pb.current_operation)
```

Comment:
Good call! I'll implement an `Operation` class.

For now, I'll just put it in `cluster.py` since none of the other services ever produce a long-running operation.